tests/acceptance/features/webUIDeleteFilesFolders/deleteFilesFolders.feature

```diff
@@ -58,17 +58,17 @@ Feature: deleting files and folders
   @smokeTest @issue-4582 @disablePreviews
   Scenario: Delete multiple files at once
     Given user "Alice" has uploaded file "data.zip" to "data.zip"
-    And user "Alice" has created file "lorem.txt"
-    And user "Alice" has created folder "simple-folder"
+    And user "Alice" has created file "file1.txt"
```

Comment:
@individual-it I am not able to reproduce the flaky test. So from what you have mentioned about deadlocks and @phil-davis mentioning about timing of the test in https://github.com/owncloud/web/issues/5439, I changed the resource name to see if we can avoid any possible deadlock. What do you think?